credentials/apps/badges/models.py

```diff
@@ -88,11 +89,12 @@ class BadgeTemplate(AbstractCredential):
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, help_text=_("Unique badge template ID."))
     name = models.CharField(max_length=255, help_text=_("Badge template name."))
     description = models.TextField(null=True, blank=True, help_text=_("Badge template description."))
-    icon = models.ImageField(upload_to="badge_templates/icons", null=True, blank=True)
+    icon = models.ImageField(upload_to="badge_templates/icons", null=True, blank=True, max_length=255)
```

Comment:
@NiedielnitsevIvan we need this in order to save accredible logo, because it has long URL (+-200)